docs/demos/opc-thermometer-demo.md

```diff
@@ -18,11 +18,16 @@ The demo consists of the following components:
 2. Agent sees the OPC UA Configuration, discovers the servers specified in the Configuration, and creates an Instance for each server.
 3. The Akri Controller sees the Instances in etcd and schedules an OPC UA Monitoring broker pod for each server.
 4. Once the OPC UA Monitoring broker pod starts up, it will create an OPC UA Client that will create a secure channel with its server.
-5. The OPC UA Client will subscribe to the OPC UA Variable with the NodeID with `Identifier` "Thermometer\_Temperature" and `NamespaceIndex` 2 as specified in the OPC UA Configuration. The server will publish any time the value of that variable changes.
+5. The OPC UA Client will subscribe to the OPC UA Variable with the NodeID with `Identifier` "FastUInt1" and `NamespaceIndex` 2 as specified in the OPC UA Configuration. The server will publish any time the value of that variable changes.
```

Comment:
Why is the identifier `FastUInt1`?

Reply:
@kate-goldenring `FastUInt1` is the identifier of the [fast changing node](https://github.com/Azure-Samples/iot-edge-opc-plc#slow-and-fast-changing-nodes:~:text=Slow%20and%20fast%20changing%20nodes) that is provided by the OPC PLC server. We could create a node with Identifier "Thermometer_Temperature" but that node would be simulated as described [here](https://github.com/Azure-Samples/iot-edge-opc-plc#slow-and-fast-changing-nodes:~:text=Please%20note%20that%20nodes%20specified%20in%20the%20JSON%20file%20are%20NOT%20part%20of%20the%20simulation). Since we can configure the range and randomness of the fast/slow changing node, `FastUInt1` is the closest node to simulate data that is similar to the OPC UA demo.

Reply:
Thanks! Maybe we add a footnote about this in case others are curious or fine to ignore

Reply:
added [footnote](https://github.com/project-akri/akri-docs/pull/56/commits/99125d5c7a9beafef826f83ad554ecb1589e9a9b).